src/internal/Popover.js

```diff
@@ -437,13 +462,15 @@ class Popover extends Component<DefaultProps, Props, void> {
           role={role}
           onRequestTimeout={this.handleRequestTimeout}
           transitionAppear
+          ref={el => (this.transitionEl = el)}
```

Comment:
we use the wording `node` on the rest of the codebase for a ref callback argument. Can we stick to that convention?